web/app/templates/user_preferences.html

```diff
@@ -102,6 +102,16 @@ <h3>Pushbullet</h3>
             {% endif %}
           </div>
         </div>
+        <br />
+        {% if bot_name %}
+        <h3>Telegram</h3>
```

Comment:
any reason why we split telegram-related templates in 2 places? either we move this block all to `web/app/templates/telegram_button.html `, or we `web/app/templates/telegram_button.html ` to here.

Reply:
Got it. That was mostly for historical reasons. Will move all the telegram stuff in here.